shell/platform/android/io/flutter/plugin/common/FlutterBasicMessageChannel.java

```diff
@@ -20,31 +18,30 @@
  * counterpart of this channel on the Dart side. The static Java type of messages sent and received
  * is Object, but only values supported by the specified {@link MessageCodec} can be used.
  *
- * The channel supports basic message send/receive operations, handling incoming method
- * invocations, and emitting event streams. All communication is asynchronous.
+ * The channel supports basic message send/receive operations. All communication is asynchronous.
  *
  * Identically named channels may interfere with each other's communication.
  */
-public final class FlutterMessageChannel<T> {
+public final class FlutterBasicMessageChannel<T> {
```

Comment:
Since this is in an `io.flutter` package already, do we need to call this a `FlutterBasicMessageChannel` instead of just a `BasicMessageChannel`. Unlike Objective-C, we can namespace these things.

Don't have a strong opinion about this. Just curious.